source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -50,18 +50,63 @@ class GeocodeJson(AbstractAutocomplete):
     TYPE_STREET = "street"
 
     def __init__(self, **kwargs):
-        self.external_api = kwargs.get('host')
+        self.host = kwargs.get('host')
         self.timeout = kwargs.get('timeout', 10)
 
+    @staticmethod
+    def call_bragi(url, method, **kwargs):
+        try:
+            return method(url, **kwargs)
+        except requests.Timeout:
+            logging.getLogger(__name__).error('autocomplete request timeout')
+            raise TechnicalError('external autocomplete service timeout')
+        except:
+            logging.getLogger(__name__).exception('error in autocomplete request')
+            raise TechnicalError('impossible to access external autocomplete service')
+
+    @classmethod
+    def _check_response(cls, response, uri):
+        if response == None:
+            raise TechnicalError('impossible to access autocomplete service')
+        if response.status_code == 404:
```

Comment:
I don't think it is an invalid argument, it's just that we don't know it :wink: 

aren't there an UnknownObject error (or something like this) ?